api-reference/v2/tables/versioning.mdx

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I wonder if we shouldn't move this to the Tutorials section  and beef it up with an example workflow of how to use the API to poll for changed data? The basic concepts are explained here, but would be great to tie it together with example call sequence as well.

Reply:
Yeah agreed it could definitely be fleshed out with some examples/tutorial. I think we could do that as a follow on though.